packages/popover/test/visual/lumo/popover.test.js

```diff
@@ -0,0 +1,47 @@
+import { fixtureSync, nextRender, nextUpdate } from '@vaadin/testing-helpers';
+import { visualDiff } from '@web/test-runner-visual-regression';
+import '@vaadin/vaadin-lumo-styles/test/autoload.js';
+import '../../not-animated-styles.js';
+import '../../../theme/lumo/vaadin-popover.js';
+
+describe('popover', () => {
+  let div, target, element;
+
+  beforeEach(async () => {
+    element = fixtureSync('<vaadin-popover></vaadin-popover>');
+    element.renderer = (root) => {
+      root.textContent = 'Content';
```

Comment:
Should it be some more dynamic content like html or something rendered content? If I remember correctly the popover is meant to be created for "rich content", so testing it with rich content for the screenshots could be beneficial to catch early problems

Reply:
Generally, visual tests aim to cover the theme styles of the component (in case of popover - its overlay offset, shadow etc). I don't think we should include other components in this test, as we don't do that e.g. in dialog.

Reply:
I was just thinking about simple things like paragraph or headings - just in case so that it doesn't look off. But I totally understand your point!